concordium-node/src/configuration.rs

```diff
@@ -322,6 +322,63 @@ pub struct RpcCliConfig {
     pub rpc_server_token: String,
 }
 
+#[derive(StructOpt, Debug)]
+/// Parameters related to the V2 GRPC interface.
+pub struct GRPC2Config {
+    #[structopt(
+        name = "grpc2-listen-addr",
+        long = "grpc2-listen-addr",
+        requires = "grpc2-listen-port",
+        help = "Address the GRPC server should listen on, e.g., 127.0.0.1",
+        env = "CONCORDIUM_NODE_GRPC2_LISTEN_ADDRESS"
+    )]
+    pub listen_addr:      Option<std::net::IpAddr>,
+    #[structopt(
+        name = "grpc2-listen-port",
+        long = "grpc2-listen-port",
+        requires = "grpc2-listen-addr",
+        help = "Address the GRPC server should listen on, e.g. 11000",
+        env = "CONCORDIUM_NODE_GRPC2_LISTEN_PORT"
+    )]
+    pub listen_port:      Option<u16>,
+    #[structopt(
+        long = "grpc2-x509-cert",
+        help = "Certificate used to enable TLS support for the GRPC V2 server.",
+        env = "CONCORDIUM_NODE_GRPC2_X509_CERT",
+        requires = "grpc2-listen-addr",
+        requires = "grpc2-cert-private-key"
+    )]
+    pub x509_cert:        Option<PathBuf>,
+    #[structopt(
+        long = "grpc2-cert-private-key",
+        help = "Private key corresponding to the certificate",
+        env = "CONCORDIUM_NODE_GRPC2_CERT_PRIVATE_KEY",
+        requires = "grpc2-listen-addr",
+        requires = "grpc2-x509-cert"
+    )]
+    pub cert_private_key: Option<PathBuf>,
+    #[structopt(
+        long = "grpc2-enable-grpc-web",
+        help = "Enable support for GRPC-Web protocol.",
+        env = "CONCORDIUM_NODE_GRPC2_ENABLE_GRPC_WEB",
+        requires = "grpc2-listen-addr"
+    )]
+    pub enable_grpc_web:  bool,
+    #[structopt(
+        long = "grpc2-endpoint-config",
+        help = "Configuration file for endpoints, listing which endpoints should be enabled or \
+                disabled.",
```

Comment:
We should mention that if the config is omitted, all endpoints will be enabled.

Reply:
Done